seed/main/bho.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE ldml SYSTEM "../../common/dtd/ldml.dtd">
+<!-- Copyright © 1991-2022 Unicode, Inc.
+For terms of use, see http://www.unicode.org/copyright.html
+SPDX-License-Identifier: Unicode-DFS-2016
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+-->
+<ldml>
+		<identity>
+			<version number="$Revision$"/>
+			<language type="bho"/>
+			<script type="Deva"/>
```

Comment:
The identity element should only reflect items in the locale ID of the file, so delete the `<script type="Deva"/>`